packages/create-subscription/package.json

```diff
@@ -2,6 +2,8 @@
   "name": "create-subscription",
   "description": "utility for subscribing to external data sources inside React components",
   "version": "16.12.0",
+  "main": "index.js",
+  "module": "index.mjs",
```

Comment:
Better to not use mjs extension at all. Webpack treats module field and mjs differently. Mixing them produce broken output.

Reply:
The approach has changed, this PR is using Node's conditional exports since node does not use the "module" field at all. This should avoid issues you have webpack's usage of the "module" field.